Cargo.toml

```diff
@@ -66,15 +82,46 @@ ibc-core-commitment-types   = { version = "0.56", default-features = false }
 ibc-core-handler-types      = { version = "0.56", default-features = false }
 ibc-client-tendermint-types = { version = "0.56", default-features = false }
 
-alloy             = { version = "0.7", default-features = false }
-alloy-contract    = { version = "0.7", default-features = false }
-alloy-sol-types   = { version = "0.8", default-features = false }
-alloy-primitives  = { version = "0.8", default-features = false }
+alloy                  = { version = "0.7", default-features = false }
+alloy-contract         = { version = "0.7", default-features = false }
+alloy-sol-types        = { version = "0.8", default-features = false }
+alloy-primitives       = { version = "0.8", default-features = false }
+alloy-trie             = { version = "0.5", default-features = false }
+alloy-serde            = { version = "0.7", default-features = false }
+alloy-rlp              = { version = "0.3", default-features = false }
+alloy-rpc-types-eth    = { version = "0.7", default-features = false }
+alloy-rpc-types-beacon = { version = "0.7", default-features = false }
+alloy-rpc-types-engine = { version = "0.7", default-features = false }
 
 sp1-sdk    = { version = "3.3", default-features = false }
 sp1-zkvm   = { version = "3.3", default-features = false }
 sp1-helper = { version = "3.3", default-features = false }
 
+cosmwasm-schema = { version = "2.1", default-features = false }
+cosmwasm-std    = { version = "2.1", default-features = false }
+cw-storage-plus = { version = "2.0", default-features = false }
+
+# The dependencies below are maintained by Sigma Prime (for use in Lighthouse (and the broader Ethereum ecosystem))
+ethereum_ssz        = { version = "0.8", default-features = false }
+ethereum_ssz_derive = { version = "0.8", default-features = false }
+tree_hash_derive    = { version = "0.8", default-features = false }
+
+# The dependencies below are maintained by Parity Tech
+trie-db            = { version = "0.29", default-features = false }
+hash-db            = { version = "0.16", default-features = false }
+memory-db          = { version = "0.32", default-features = false }
+hash256-std-hasher = { version = "0.15", default-features = false }
+rlp                = { version = "0.6", default-features = false }
+rlp-derive         = { version = "0.2", default-features = false }
+primitive-types    = { version = "0.13", default-features = false }
+
+# From union (might be removed if they are replaced by something more standard)
+typenum = { version = "1.17.0", default-features = false }
+
+# dev-dependencies
+cw-multi-test = { version = "2.2.0", default-features = false }
+milagro_bls   = { git = "https://github.com/Snowfork/milagro_bls", rev = "bc2b5b5e8d48b7e2e1bfaa56dc2d93e13cb32095", default-features = false } # Only used for testing, not to be used in production!
+
 [patch.crates-io]
```

Comment:
Yeah, seems like it is not possible. We should be very sure that this will not create any issues because it seems less than ideal to have to pin to these patches for all packages here :/